libs/designsystem/header/src/header.component.html

```diff
@@ -0,0 +1,22 @@
+
+<div class="container" [ngClass]="{ centered: centered }">
+  <div *ngIf="!!avatar" class="avatar">
+    <ng-content select="kirby-avatar"></ng-content>
+  </div>
+  <div *ngIf="!!flag" class="flag">
+    <ng-content select="kirby-flag"></ng-content>
+  </div>
+  <h1 *ngIf="!!title && !!value" class="key kirby-text-medium">
```

Comment:
Since the title has the same bottom margin whether it's rendered small or not, I don't see a need for a separate css class:
```suggestion
  <h1 *ngIf="!!title && !!value" class="title kirby-text-medium">
```